src/plugins/xfconf/README.md

```diff
@@ -13,9 +13,38 @@
 
 This is a storage plugin to mount the xfconf configuration settings.
 
+## Xfconf Terminology
+
+### Property
+
+A property in xfconf is the same as a key in libelektra i.e. it has a name and can hold one or more values.
+
+### Channel
+
+A channel is a type of namespace used in the xfconf library.
+Usually, it is used to separate the properties of different applications which is helpful if different applications rely on a property with the same name but require them to hold different values.
+For example Thunar uses a channel named `thunar`, Xfwm uses a channel named `xfwm4` and so on.
+Keep in mind that channels are only used to separate the properties such as namespaces.
+They are not a security feature i.e. every application has read/write access to every channel.
+
+The list of all channels is stored in the `system:/elektra/modules/xfconf/channels` which is an array of all channel names.
+Channel cannot be explicitly created or removed.
+They only exist in an implicit manner when you pass the `channel=...` argument.
+
+### Locks
+
+Xfconf uses so-called locks to prevent users or applications to set properties to a new value.
+They can be referred as a type of constant.
+As of now, locks cannot be set or unset in xfconf which makes locked properties read-only forever.
+
 ## Dependencies
 
-xfconf from the XFCE project
+The xfconf library from the XFCE project is the main dependency of this plugin.
+Usually, this library is called something such as `xfconf` (Arch, Fedora, `xfconf-devel` for compiling), `libxfconf-0` (Debian, `libxfconf-0-dev` for compmiling) or `xfce4-conf` (FreeBSD) in the package manager.
+As xfconf itself depends on dbus and glib, these are dependecies too but should be installed with the package manager automatically.
```

Comment:
Please also write that dbus must be running.

Reply:
Done with 1b1307a74539c4bbd6c0cc9da3c9903e3db7e53c.